client/src/App.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import ReviewService from './components/reviewService.jsx';
+import querystring from 'querystring';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      courseId: null
```

Comment:
May be a bit unnecessary to define courseID as null, line 25 will still evaluate to undefined if you leave this out of your constructor